Analysis/Tasks/PWGCF/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@
 # granted to it by virtue of its status as an Intergovernmental Organization or
 # submit itself to any jurisdiction.
 
+add_subdirectory(o2femtodream)
```

Comment:
why? There is AliFemtoDream and o2femtodream. I like the nomenclature and want to keep it

Reply:
No class or library in O2 has an extra o2 in it. Please remove it. It will just be FemtoDream which sounds nice, doesn't it? :)

Reply:
ok, but this is how we internally decided to call our framework, also to make the distinction to the existing FemtoDream in AliPhysics clear.

Reply:
You can still of course call it like that internally, but the directory should not include not (neither the class names, but I think they do not). Please change the directory name so that we can move ahead.

Reply:
I don't understand this but whatever.